index.html

```diff
@@ -0,0 +1,1378 @@
+                        </path>
+                    </g>
+                </svg>
+                <p>Communities</p>
+            </div>
+            <div class="nav-link icons">
+                <svg width="26.25" height="26.25" viewBox="0 0 24 24" aria-hidden="true" class="icons-left">
+                    <g>
+                        <path
+                            d="M18.244 2.25h3.308l-7.227 8.26 8.502 11.24H16.17l-5.214-6.817L4.99 21.75H1.68l7.73-8.835L1.254 2.25H8.08l4.713 6.231zm-1.161 17.52h1.833L7.084 4.126H5.117z">
+                        </path>
+                    </g>
+                </svg>
+                <p>Premium</p>
+            </div>
+            <div class="nav-link icons">
+                <svg width="26.25" height="26.25" viewBox="0 0 24 24" aria-hidden="true" class="icons-left">
+                    <g>
+                        <path
+                            d="M5.651 19h12.698c-.337-1.8-1.023-3.21-1.945-4.19C15.318 13.65 13.838 13 12 13s-3.317.65-4.404 1.81c-.922.98-1.608 2.39-1.945 4.19zm.486-5.56C7.627 11.85 9.648 11 12 11s4.373.85 5.863 2.44c1.477 1.58 2.366 3.8 2.632 6.46l.11 1.1H3.395l.11-1.1c.266-2.66 1.155-4.88 2.632-6.46zM12 4c-1.105 0-2 .9-2 2s.895 2 2 2 2-.9 2-2-.895-2-2-2zM8 6c0-2.21 1.791-4 4-4s4 1.79 4 4-1.791 4-4 4-4-1.79-4-4z">
+                        </path>
+                    </g>
+                </svg>
+                <p>Profile</p>
+            </div>
+            <div class="nav-link icons">
+                <div class="icons">
+                    <svg width="26.25" height="26.25" viewBox="0 0 24 24" aria-hidden="true" class="icons-left">
+                        <g>
+                            <path
+                                d="M3.75 12c0-4.56 3.69-8.25 8.25-8.25s8.25 3.69 8.25 8.25-3.69 8.25-8.25 8.25S3.75 16.56 3.75 12zM12 1.75C6.34 1.75 1.75 6.34 1.75 12S6.34 22.25 12 22.25 22.25 17.66 22.25 12 17.66 1.75 12 1.75zm-4.75 11.5c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25S6 11.31 6 12s.56 1.25 1.25 1.25zm9.5 0c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25-1.25.56-1.25 1.25.56 1.25 1.25 1.25zM13.25 12c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25.56-1.25 1.25-1.25 1.25.56 1.25 1.25z">
+                            </path>
+                        </g>
+                    </svg>
+                </div>
+                <div>
+                    <p>More</p>
+                </div>
+            </div>
+
+            <div class="post-button ">
+                <button>
+                    <p>Post</p>
+                </button>
+            </div>
+
+            <div class="spacer"></div>
+            <div class="spacer nav-link"></div>
+            <!-- <div class="spacer nav-link"></div> -->
+
+            <div class="user">
+                <div class="profile-pic">
+                    <img src="./svgFiles/Hello Ctulhu! black letters.jpg" alt="" width="45px">
```

Comment:
I would avoid having spaces and special characters (!) in file names, they can cause unexpected issues on certain systems. Better to go with underscores and dashes just to be safe